### PR TITLE
Release 8: Add showing error message when test was already finished

### PR DESCRIPTION
**Pull request by mBeym**
Mantis Ticket: https://mantis.ilias.de/view.php?id=36512

- Clicking on the `Finish Test`, `Next question` or `Previous question` button will now show an error message if the user has no unfinished passes currently (meaning it was already finished from somewhere/someone else.

### CODE REVIEW

**Comment by dsstrassner** [comment]
Hi @matthiaskunkel 


please have a look at the new lang_var and the introduced text. This fix should remove 1 failing Test case 😉

@mbecker-databay / @kergomard please wait for the feedback and approval from @matthiaskunkel

**Comment by mBeym** [comment]
@dsstrassner lang fixed :)

File: lang/ilias_de.lang [hunk 1]
```diff
@@ -1372,6 +1372,7 @@ assessment#:#tst_count_correct_solutions#:#Nur vollständig richtig beantwortete
 assessment#:#tst_count_correct_solutions_desc#:#Vollständige und richtige Antworten erhalten die maximale Punktzahl. In allen anderen Fällen gibt es 0 Punkte. Dies gilt auch für Fragen, die Punkte für Teilantworten vorsehen.
 assessment#:#tst_count_partial_solutions#:#Auch teilweise falsch beantwortete Fragen ergeben Punkte
 assessment#:#tst_count_partial_solutions_desc#:#Bei einigen Fragetypen können für korrekt gegebene Teilantworten Punkte erzielt werden. Diese werden aufsummiert. Dies betrifft die Fragetypen Multiple Choice, KPrim, Mehrfachauswahl in Imagemaps, Fehler / Worte markieren und Zuordnungsfrage.
+assessment#:#tst_current_pass_no_longer_valid#:#Ihr aktueller Test Durchlauf ist nicht mehr gültig. <br>Er wurde wahrscheinlich bereits abgeschlossen oder von einem Tutor beendet.
```
Comment (matthiaskunkel) [inline lang/ilias_de.lang:1375]: Testdurchlauf, nicht Test Durchlauf

File: lang/ilias_en.lang [hunk 2]
```diff
@@ -1372,6 +1372,7 @@ assessment#:#tst_count_correct_solutions#:#Only Correct and Complete Answers Sco
 assessment#:#tst_count_correct_solutions_desc#:#Participants score either the maximum number of points for a fully correct and complete answer or 0 points in every other case. This is also valid for questions that define points for partial answers.
 assessment#:#tst_count_partial_solutions#:#Incomplete or Partly Wrong Answers Score Points
 assessment#:#tst_count_partial_solutions_desc#:#When a question defines points for partial answers, points for correctly provided partial answers are added up: Participants can score points for incomplete or partly wrong answers on these questions.
+assessment#:#tst_current_pass_no_longer_valid#:#Your current Test Pass is no longer valid. <br>It was probably already completed or finished by a Tutor.
```
Comment (matthiaskunkel) [inline lang/ilias_en.lang:1375]: According to our English expert Chris Potter, the term 'pass' should only be used for a successful test run (I still would like to streamline this until 8.0 stable). The use of 'pass' in this lang var shows the problem. The test run is not even completed but called pass. That's confusing for a native speaker. Better would be: 'Your current test run is no longer valid. <br>It was probably completed already or finished by a tutor.' And neither 'test run' nor 'Tutor' needs capital letters. 'Tutor' in capital letters would refer to the role 'Tutor', while 'tutor' is a general term for those people who are tutoring...

**Review by matthiaskunkel — APPROVED** [review]
Thanks @mbeym for the fixes. Can be merged now.

**Pull request merged**